src/components/organisms/artefact-overview/artefact-overview.jsx

```diff
@@ -62,7 +62,8 @@ const ArtefactOverview = ({
         { ListView === view && <ArtefactLine
             title={ item.title }
             subtitle={ item.subtitle }
-            text={ item.text }
+            date={item.date}
+            masterData={item.masterData}
```

Comment:
Bezüglich `masterData` könnte man sich überlegen, ob die Ausgabe von masterData in der ArtifactLine nicht bereits außerhalb der Komponente vorbereiten kann und die Elemente einfach als Attribut / über ein Parameter reinreicht. Denn so wird die ArtifactLine spezifischer, da sie nun von der Struktur von masterData abhängt.
Ändert sich somit etwas an der Struktur von masterData, muss dann auch die ArtifactLine angefasst werden.